UnityProject/Assets/Scripts/Objects/Furniture/BuckleInteract.cs

```diff
@@ -133,6 +140,7 @@ public void BucklePlayer(PlayerScript playerScript)
 			}
 
 			objectPhysics.BuckleObjectToThis(playerScript.playerMove);
+			OnBuckleEvent?.Invoke();
```

Comment:
This is a nitpick, but you can ignore this one if you'd like; Events are best invoked last when all the logic above it has passed first. In this scenario there _shouldn't_ be an issue per say, but since things are being done in `occupiedSpriteHandler` this might override other stuff the events are trying to do in the future.